manifest.yml

```diff
@@ -7,3 +7,4 @@ applications:
     env:
       USERNAME: ((APP_USERNAME))
       PASSWORD: ((APP_PASSWORD))
+      ENABLE_MULTIPLE_CHOICE_ANSWER_TYPE: ((ENABLE_MULTIPLE_CHOICE_ANSWER_TYPE))
```

Comment:
you'll need to update the deploy action also - given this is just for CSL i think we can just manually set it tho rather than require it in the manifest, so we can remove this